### ISSUE TITLE
Change the DB structure - use a table for translation

### ISSUE THREAD

**Issue by gagarine** [BoltTranslate/Translate#184]
Instead of serializing data why not creating a new table for translatable field for each content type?

![screenshot 2018-10-17 at 12 21 45](https://user-images.githubusercontent.com/164074/47081022-d3844600-d209-11e8-9aab-8bd981effeb8.png)

I personally things at least a DB structure than allow translation should be supported in core and opened the discussion their : https://discuss.bolt.cm/d/474-multi-language-roadmap/2 . But this plugin can be a great way to experiment.

**Comment by SahAssar** [comment]
Well, at the outset of this one of the goals were to not have to change the db structure when adding a field as translatable, so that restricted the implementation. We have also had a number of problems that don't fit neatly into that model like repeaters, blocks, templatefields and even worse stuff like repeaters-in-templatefields.

My work on this extension came out of necessity, so it's not at all nice-and-tidy. If you feel like improving it either just submit a PR or talk to me or bob (or wait for bolt v4, since it'll probably be easier to have a proper solution there).

**Comment by gagarine** [comment]
Yeah, I understand Bolt a bit more now and you choices make sens. I think any big change should wait the next version of Bolt. Apparently they are going to change the data structure.
